src/AccentHeader.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+export const ALIGN_VALUES = ['left', 'right', 'center'];
+
+const AccentHeader = ({
+	align,
+	headingTag,
+	children,
+	className,
+	...other
+}) => {
+	const HeadingTagEl = headingTag;
+	return (
+		<header className="accentHeaderWrapper">
+			<HeadingTagEl
+				className={cx(
+					`accentHeader
+					accentHeader--${align}
+					align--${align}
+					text--normal`,
+					className
+				)}
+			>
+				{children}
+			</HeadingTagEl>
+		</header>
+	);
+};
+
+AccentHeader.defaultProps = {
+	align: 'left',
+	headingTag: 'h2'
+};
+
+PropTypes.AccentHeader = {
+	align: PropTypes.oneOf(ALIGN_VALUES),
+	headingTag: PropTypes.string
```

Comment:
suggestion:
```
headingTag: PropTypes.oneOf([
'h1', 'h2', 'h3', 'h4', 'h5', 'h6'
])
```

Reply:
I thought about doing that, but I thought it was cool to let them pass whatever they want. However, I now realize that's a bad idea if these are nested in a `<header>`